openedx/core/djangoapps/catalog/tests/test_utils.py

```diff
@@ -172,6 +174,51 @@ def test_config_missing(self, _mock_cache, _mock_get_catalog_data):
         data = utils.get_programs(self.user)
         self.assertEqual(data, [])
 
+    def _expected_progam_credentials_data(self):
+        """
+        Dry method for getting expected program credentials response data.
+        """
+        return [
+            credentials_factories.UserCredential(
+                id=1,
```

Comment:
I don't think you need to specify this id, the factory should be self-incrementing this. Try it out and see if it works.